doc/source/parameters/unresolved-cfd-dem/cfd-dem.rst

```diff
@@ -6,17 +6,19 @@ This subsection includes parameters related to multiphase flow simulations using
 .. code-block:: text
 
    subsection cfd-dem
-      set grad div = false
-      set void fraction time derivative = false
+      set grad div = true
+      set void fraction time derivative = true
       set interpolated void fraction = true
+      set vans model = modelA
       set drag force = true
+      set drag model = dallavalle
```

Comment:
I think the default one should be Di Felice, it's the most common IMO, what do you think?

Reply:
I agree. Just changed it.